File: 171-excel-sheet-column-number/excel-sheet-column-number.cs
public class Solution {
    public int TitleToNumber(string columnTitle) {
        int ans=0;
        int doubler=0;
        for(int i=columnTitle.Length-1;i>-1;i--){
            ans+=(columnTitle[i]-'A' + 1)*(int)Math.Pow(26,doubler);
            doubler+=1;
        }
        return ans;
    }
}

File: 171-excel-sheet-column-number/excel-sheet-column-number.java
class Solution 
{
    public int titleToNumber(String columnTitle) 
    {
        int len = columnTitle.length();
        int sum = 0;
        int value = 0;

        for (int i = 0; i < len; i++) 
        {
            char c = columnTitle.charAt(i);
            value = c - 'A' + 1;
            sum = sum * 26 + value;
        }
        return sum;
    }
}